test/testdata/golden/gapic/lang/csharp_gapic.yaml

```diff
@@ -0,0 +1,5 @@
+type: com.google.api.codegen.ConfigProto
```

Comment:
I am still hesitant to check in so many static files for this test, especially since they may need to be maintained as toolkit is updated. I would prefer to check in the test artman yaml to `googleapis` and submodule it, but that doesn't need to be done in this PR.